CHANGES.md

```diff
@@ -9,6 +9,10 @@ Change Log
     * `Viewer.terrainShadows` now uses the `ShadowMode` enum instead of a Boolean, e.g. `viewer.terrainShadows = ShadowMode.RECEIVE_ONLY`.
 * Removed the default gamma correction for Bing Maps aerial imagery, because it is no longer an improvement to current versions of the tiles.  To restore the previous look, set the `defaultGamma` property of your `BingMapsImageryProvider` instance to 1.3.
 * Fixed billboard rotation when sized in meters. [#3979](https://github.com/AnalyticalGraphicsInc/cesium/issues/3979)
+* Added `DebugCameraPrimitive` to visualize the view frustum of a camera.
```

Comment:
Not sure why these changes are in this branch.  Perhaps merge in master and make sure only your new entry is here.